eng/build.yml

```diff
@@ -24,7 +24,9 @@ jobs:
     artifacts:
       publish:
         logs: true
-        manifests: true
+        ${{ if ne(variables['System.TeamProject'], 'public') }}:
+          artifacts: true
```

Comment:
artifacts too? Do you need anything outside the packages?

Reply:
When setting publish.artifacts = true, it will pull out the files from artifacts/bin and artifacts/packages and publish them. It'll be nice to have the binaries published so that when we add tests, we could add a testing stage (which would allow reruns for the tests in case there is anything flaky, instead of having to rerun the entire pipeline or the build stage) to pull down the binaries and packages and run them.